### PR TITLE
Automate Fern publication

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: api/fern/apis/iam/openapi/openapi.yaml [hunk 1]
```diff
@@ -199,7 +199,6 @@ components:
           enum:
           - DEPLOYMENT_ADMIN
           - WORKSPACE_OWNER
```
Comment (jwitz) [inline api/fern/apis/iam/openapi/openapi.yaml:201]:
```suggestion
```

File: api/fern/apis/iam/openapi/openapi.yaml [hunk 2]
```diff
@@ -681,12 +679,11 @@ components:
       - role
       - workspaceId
       type: object
-  securitySchemes:  
-    Bearer:
-      type: http
-      scheme: bearer
-security: 
-  - Bearer: []
```
Comment (dsinghvi) [inline api/fern/apis/iam/openapi/openapi.yaml:689]: lets remove this and we can come back to it in a separate PR

File: api/fern/apis/iam/openapi/openapi.yaml [hunk 3]
```diff
@@ -681,12 +679,11 @@ components:
...
+  securitySchemes:
+    JWT:
+      in: header
+      name: Authorization
+      type: apiKey
```
Comment (jwitz) [inline api/fern/apis/iam/openapi/openapi.yaml:686]:
```suggestion
  securitySchemes:  
    Bearer:
      type: http
      scheme: bearer
security: 
  - Bearer: []
```

File: api/fern/apis/iam/openapi/openapi.yaml [hunk 4]
```diff
@@ -198,8 +198,6 @@ components:
           description: The role of the API token.
           enum:
           - DEPLOYMENT_ADMIN
-          - WORKSPACE_OWNER
-          - WORKSPACE_OPERATOR
           - WORKSPACE_AUTHOR
```
Comment (jwitz) [inline api/fern/apis/iam/openapi/openapi.yaml:201]:
```suggestion
          - WORKSPACE_OWNER
          - WORKSPACE_OPERATOR
          - WORKSPACE_AUTHOR
```

File: .github/workflows/apipublish.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+name: publish-docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'api/**'
```
Comment (dsinghvi) [inline .github/workflows/apipublish.yml:8]: path to the existing github workflow should be added here

File: .github/workflows/apipreview.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+name: preview-docs
+
+on: 
+  pull_request:
+    paths:
+    - 'api/**'
```
Comment (jwitz) [inline .github/workflows/apipreview.yml:6]:
```suggestion
    - 'api/**'
    - '.github/workflows/**'
```

**Pull request merged**